torch/_inductor/codegen/cpp.py

```diff
@@ -3486,6 +3365,41 @@ def _select_tiling_indices(
             return contig_vars_sorted
         return sorted(contig_vars_sorted, key=contig_vars_list.count)[-1:]
 
+    @staticmethod
+    def _dtype_not_supported(loop_bodies, supported_dtype):
```

Comment:
Is it better to factor out a function that returns a set of all the used dtypes in the loop bodies and check the supported dtypes on the returned value? Then, we can perhaps replace `get_loop_body_lowp_fp` with it too?

Reply:
Hi, @jgong5 . From my understanding, `get_loop_body_lowp_fp` is not only check the dtype but also check  whether the `_node.target` is in `[load, store, bas, neg, output]`.  So maybe we can leave them as two util fuctions?